python/openassetio/test/manager/_implementation.py

```diff
@@ -14,7 +14,9 @@
 #   limitations under the License.
 #
 """
-Boilerplate classes for initialization and execution of test cases.
+@namespace openassetio.test.manager._implementation
+Private implementation classes for the manager test framework.
+@private
```

Comment:
[question] The `@private` tag on the class/function below means the docstring is not rendered by Doxygen. The `@private` on this `@namespace` seems to have no effect, i.e. `_implementation` is listed in the docs for the `manager` package, but appears nowhere else. Is that all intentional?

Reply:
I was trying to hide it as much as possible. I wondered If it was worth leaving there to make sure it was obvious to people reading the src too?

Reply:
Yeah that's fine, just wanted to ensure it's intentional.